deepchem/feat/__init__.py

```diff
@@ -25,3 +25,13 @@
 from deepchem.feat.material_featurizers import ElementPropertyFingerprint
 from deepchem.feat.material_featurizers import SineCoulombMatrix
 from deepchem.feat.material_featurizers import CGCNNFeaturizer
+
+try:
+    import transformers
+    from transformers import BertTokenizer
+
+    from deepchem.feat.smiles_tokenizer import SmilesTokenizer
+    from deepchem.feat.smiles_tokenizer import BasicSmilesTokenizer
+except ImportError:
```

Comment:
I think you might need `ModuleNotFoundError` here instead of `ImportError`. See this example import guard: https://github.com/deepchem/deepchem/blob/master/deepchem/models/__init__.py#L40

Reply:
done